pkg/vm/vm.go

```diff
@@ -910,6 +910,15 @@ func (v *VM) execute(ctx *Context, op opcode.Opcode, parameter []byte) (err erro
 
 		v.estack.PushVal(new(big.Int).Rem(a, b))
 
+	case opcode.POW:
+		exp := v.estack.Pop().BigInt()
```

Comment:
What if `a == 1` and `exp > (2³¹ -  1)`?

Reply:
Missed that, fixed.